src/ApiService/ApiService/onefuzzlib/VmssOperations.cs

```diff
@@ -262,6 +262,8 @@ private Task<string> GetInstanceIdForVmId(Guid scaleset, Guid vmId)
             return OneFuzzResultVoid.Ok;
         } catch (RequestFailedException ex) when (ex.Status == 409 && ex.Message.StartsWith("The request failed due to conflict with a concurrent request")) {
             return OneFuzzResultVoid.Error(ErrorCode.UNABLE_TO_UPDATE, $"protection policy update is already in progress: {vmId}:{instanceId} in vmss {scaleset.ScalesetId}");
+        } catch (RequestFailedException ex) when (ex.Status == 400 && ex.Message.Contains("The provided instanceId") && ex.Message.Contains("not an active Virtual Machine Scale Set VM instanceId.")) {
```

Comment:
This will still get logged as an error right ? i think you have to update the call to this in `AcquireScaleInProtection` and `ReleaseScaleInProtection`

Reply:
It's going to be logged as a trace with severity level: Error aka: it won't show up in the Failures tab of app insights.

Reply:
Resolved offline. It makes sense that we don't want to log it as an exception because it's an expected error but, it's still an error that shouldn't be happening even though we're ok with it happening given our current system architecture. Removing the exception logging but keeping the trace severity level as error seems to be an OK balance. Otherwise, we can add a specific case for this exception to only log as info.